sql/migrations/0001_initial.sql

```diff
@@ -88,3 +88,13 @@ CREATE TABLE atx_top
     atx_id CHAR(32),
     layer  INT NOT NULL
 );
+
+CREATE TABLE poets
+(
+    ref        VARCHAR PRIMARY KEY,
+    poet       BLOB,
+    service_id VARCHAR,
+    round_id   VARCHAR
+) WITHOUT ROWID;
```

Comment:
WITHOUT ROWID is not good with big blobs, it reduces tree fanout that makes traversal slower
checkout [part 4](https://www.sqlite.org/withoutrowid.html)

Reply:
thanks for the link